master/buildbot/reporters/gerrit_verify_status.py

```diff
@@ -226,6 +229,16 @@ def send(self, build):
                     reporter=reporter,
                     category=category,
                     duration=duration)
+
+                # The verify-status plugin will return 204 NO CONTENT
+                # on success.
+                if response.code != http_client.NO_CONTENT:
+                    try:
+                        body = yield response.text()
```

Comment:
see: http://docs.buildbot.net/current/developer/utils.html#buildbot.util.service.IHTTPResponse
Note
IHTTPResponse is a subset of treq Response API described here The API it is voluntarily minimalistic and reflects what is tested and reliable to use with the three backends (including fake). The api is a subset of the treq API, which is itself a superset of twisted IResponse API. treq is thus implemented as passthrough.

Notably:

There is no api to automatically decode content, as this is not implemented the same in both backends.

so please use .content(), and decode manually.